src/Microsoft.TestPlatform.Extensions.TrxLogger/Utility/Converter.cs

```diff
@@ -148,6 +148,11 @@ public static TrxObjectModel.TestOutcome ToOutcome(ObjectModel.TestOutcome rockS
 
         public static List<CollectorDataEntry> ToCollectionEntries(IEnumerable<ObjectModel.AttachmentSet> attachmentSets, TestRun testRun, string trxFileDirectory)
         {
+            if (EqtTrace.IsInfoEnabled)
+            {
+                EqtTrace.Info($"Converter.ToCollectionEntries: Converting attachmentSets {string.Join(",", attachmentSets)} to collection entries.");
```

Comment:
attachmentSets null check is below and here we are printing before null check?

Reply:
Thanks. added null check.